.github/workflows/default-tests.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     - name: Python ${{ matrix.python-version }}
       shell: bash -l {0}
       run: |
-        conda create --name TEST python=${{ matrix.python-version }} pip --file requirements.txt --file test_requirements.txt --strict-channel-priority
+        conda create --name TEST python=${{ matrix.python-version }} pip "libnetcdf<4.8.0" --file requirements.txt --file test_requirements.txt --strict-channel-priority
```

Comment:
We should probably revert this change. We need to unpin libnetcdf.

Reply:
Sorry, I accidentally merged in some stuff related to 1.9 into develop, which I didn't mean to do.  Some of the commits above are my attempts to rectify some of the issues.  Others are just from merging master.  I'll unpin this.